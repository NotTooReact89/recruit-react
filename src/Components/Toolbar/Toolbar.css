.toolbar {
    position: fixed;
    top: 0;
    left: 0;
    width: 100%;
    background: #000000;
    height: 56px;
  }
  
  .toolbar__navigation {
    display: flex;
    height: 100%;
    align-items: center;
    padding: 0 1rem;
  }
