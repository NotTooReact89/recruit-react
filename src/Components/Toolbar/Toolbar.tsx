import * as React from 'react';

import { DrawerToggleButton } from '..';
import './Toolbar.css';

interface IToolbarProps {
  drawerClickHandler: () => void;
}

const Toolbar = (props: IToolbarProps) => (
  <header className='toolbar'>
    <nav className='toolbar__navigation'>
        <div className='toolbar__toggle-button'>
            <DrawerToggleButton click={props.drawerClickHandler} />
        </div>
    </nav>
  </header>
);

export default Toolbar;