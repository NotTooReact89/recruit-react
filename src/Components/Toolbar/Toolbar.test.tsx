import * as React from 'react';
import Toolbar from './Toolbar';
const renderer = require('react-test-renderer');

it('should render credit card form component correctly', () => {
  const tree = renderer.create((
    <Toolbar drawerClickHandler={jest.fn()} />
  )).toJSON()
  expect(tree).toMatchSnapshot()
})