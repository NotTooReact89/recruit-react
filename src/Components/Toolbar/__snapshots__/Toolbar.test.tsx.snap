// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render credit card form component correctly 1`] = `
<header
  className="toolbar"
>
  <nav
    className="toolbar__navigation"
  >
    <div
      className="toolbar__toggle-button"
    >
      <button
        className="toggle-button"
        onClick={[MockFunction]}
      >
        <div
          className="toggle-button__line"
        />
        <div
          className="toggle-button__line"
        />
        <div
          className="toggle-button__line"
        />
      </button>
    </div>
  </nav>
</header>
`;
